packages/lib-content/src/screens/FAQ/FAQ.js

```diff
@@ -118,17 +119,17 @@ function FAQPage() {
                       />,
                       <Anchor
                         key='FEM-github-issues'
-                        href='https://github.com/zooniverse/front-end-monorepo/issues'
+                        href='https://github.com/zooniverse'
```

Comment:
Not blocking, but if a volunteer goes to the GitHub Zooniverse org landing I'm not sure it's clear how to report a bug or make a feature request. Can we add anything to the GitHub Zooniverse org page to help direct someone? I'm not sure. Otherwise https://github.com/zooniverse/front-end-monorepo/issues/new/choose might be an option.

Reply:
That's a good point! However, I think the assumption is that if someone goes to Github with the intention of reporting a bug, then they're an experienced Github user. I'll tag in @seanmiller26 to double check though. Do we want users to go directly to the front-end-monorepo url? Change was initially suggested here: https://github.com/zooniverse/front-end-monorepo/issues/6177

Reply:
It looks like this change was initially suggested by Laura in the Copy Doc. I'm ok with leaving the URL as the base repo.